static/sass/_pattern_blog-card.scss

```diff
@@ -33,6 +33,54 @@
     flex-direction: column;
     padding: 0;
 
+    .blog-p-card__header {
+      @extend %post-card-header;
+
+      border-top: 3px solid $color-mid-dark;
+
+      &--cloud-and-server {
```

Comment:
Same as the other comment, we probably don't need this styles if we don't have such separation